### DISCUSSION TITLE
Use CVAT without import images

### DISCUSSION THREAD

**Discussion by wesleynf** [cvat-ai/cvat#3163] [General]
Hello,
   I have about 1T images but my HD don't has free space to import all images to CVAT database. I would like to know if I can use my images to annotate without import as to local CVAT database.

**Comment by nmanovic** [comment]
@wesleynf , try to attach a directory with all these images as a share to CVAT (https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/documentation/installation.md#share-path). By default, CVAT will not copy data into its container. For optimization purpose it will create a cache with data but it will have limited size.
